src/wmtk/SimplicialComplex.hpp

```diff
@@ -149,6 +149,13 @@ class SimplicialComplex
     static bool link_cond(const Mesh& m, Tuple t);
     static bool link_cond_bd_2d(const Mesh& m, Tuple t);
 
+    //////////////////////////////////
+    // link condition for edge mesh
+    // input Tuple t ---> edge (a,b)
+    // check if edge(a,b) is the only edge in EdgeMesh m
+    //////////////////////////////////
+    static bool link_cond_bd_1d(const Mesh& m, Tuple t);
```

Comment:
I like it a lot that you are adding documentation. Please use the doxygen style:
```
/**
* @brief short explanation
* long explanation if necessary
*
* @param m explain parameter "m"
* @param whatever explain parameter "whatever"
* @return true if blabla, false otherwise (or something like that)
*/
```

You can find the full documentation on Doxygen documentation [here](https://www.doxygen.nl/manual/docblocks.html)